### ISSUE TITLE
Cannot send text message with dual-sim

### ISSUE THREAD

**Issue by LeOSW42** [GSConnect/gnome-shell-extension-gsconnect#1130]
Hello,

Since I put a second SIM card, I'm not able anymore to send messages from GSConnect.
Also, I'm not able to select the SIM card to send the message, that's maybe he issue?

Léo

**Comment by techconsulter** [comment]
Same here

**Comment by andyholmes** [comment]
GSConnect does not have support for dual-sim messaging, so someone will have to contribute support for v2 of the SMS protocol for that to work. Sorry!

**Comment by FrancoRATOVOSON** [comment]
is this really under developpment ?
I have the same problem

**Comment by andyholmes** [comment]
No, any issue marked with the `<help wanted>` label is waiting for someone in the community to contribute.
